test/e2e/host_path.go

```diff
@@ -77,27 +76,40 @@ var _ = Describe("hostPath", func() {
 	It("should support r/w", func() {
 		volumePath := "/test-volume"
 		filePath := path.Join(volumePath, "test-file")
+		retryDuration := 180
 		source := &api.HostPathVolumeSource{
 			Path: "/tmp",
 		}
 		pod := testPodWithHostVol(volumePath, source)
 
 		pod.Spec.Containers[0].Args = []string{
-			fmt.Sprintf("--fs_type=%v", volumePath),
 			fmt.Sprintf("--rw_new_file=%v", filePath),
 			fmt.Sprintf("--file_mode=%v", filePath),
 		}
+
 		testContainerOutputInNamespace("hostPath r/w", c, pod, 0, []string{
 			"mode of file \"/test-volume/test-file\": -rw-r--r--",
 			"content of file \"/test-volume/test-file\": mount-tester new file",
```

Comment:
I would expect this not to work since the container isn't run with `--file-content`.